lua_examples/pcm/play_file.lua

```diff
@@ -0,0 +1,40 @@
+-- ****************************************************************************
+-- Play file with pcm module.
+--
+-- Upload jump_8k.u8 to spiffs before running this script.
+--
+-- ****************************************************************************
+
+
+function cb_drained(d)
+  print("drained "..node.heap())
+
+  file.seek("set", 0)
+  -- uncomment the following line for continuous playback
+  --d:play(pcm.RATE_8K)
+end
+
+function cb_stopped(d)
+  print("playback stopped")
+  file.seek("set", 0)
+end
+
+function cb_paused(d)
+  print("playback paused")
+end
+
+file.open("jump_8k.u8", "r")
+
+drv = pcm.new(pcm.SD, 1)
```

Comment:
@larkin just gave me a new build using the latest and missing the sigma-delta. PCM works fine and the  pcm.SD call doesn't cause an error. Thanks!